router/terraform/topics.tf

```diff
@@ -0,0 +1,19 @@
+module "ec2_terminating_topic" {
```

Comment:
We might need to talk about why these topics are necessary (or whether they are at all). We use them (along with a lambda not added here) to prevent downtime by ensuring that hosts are drained of tasks before being terminated - you might well want the same functionality.